python/lsst/validate/drp/report_performance.py

```diff
@@ -0,0 +1,158 @@
+# LSST Data Management System
+# Copyright 2017 AURA/LSST.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <https://www.lsstcorp.org/LegalNotices/>.
+
+from astropy.table import Column, Table
+
+from lsst.validate.base import load_metrics
```

Comment:
This is right. `validate_base` is the first-generation framework and is what's compatible with SQUASH. `verify` is the next-generation framework and will be compatible with a future version of SQUASH. Angelo is currently re-doing the SQUASH deployment with Kubernetes to quicken the development cycle and I think that updating the SQUASH DB/API to accept the new datasets from `verify` will be a priority in September. Once SQUASH works with `lsst.verify` we can port `validate_drp` over.